README.md

```diff
@@ -196,11 +196,16 @@ This is a schematic logging configuration to show log messages from input_nameA
 
 - `files` type - `files` output supports storing logs in the local files usually in /var/log.<br>
   **available options**
-  - `facility`: Facility; default to `*`.
-  - `severity`: Severity; default to `*`.
-  - `exclude`: Exclude list; default to none.
+  - `facility`: Facility in selector; default to `*`.
+  - `severity`: Severity in selector; default to `*`.
+  - `exclude`: Exclude list used in selector; default to none.
+  - `property`: Property in property-based filter; no default
```

Comment:
I think for the sake of consistency this should be `prop_name` - or, use `property_op` and `property_value`